### PR TITLE
V1 calls

### PR DESCRIPTION
**Pull request by svenlombaert**

### CODE REVIEW

File: src/calls.apib [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                + contact
+        + for_id: `1` (number, required) - The identifier of the contact/company (whichever one was given in `for`)
+        + description: `Call to discuss quote` (string, optional) -  description of the call.
+        + custom_field_ID: `bar` (string, optional) - Replace ID by the ID of your custom field. (eg custom_field_310)
+
++ Response 200 (text/html)
+    
+    + Body
+
+            "1234"
```
Comment (steftrenson) [inline src/calls.apib]: PP: Document that this is the id returned?

File: src/calls.apib [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
++ Response 200 (application/json)
+    
+    + Attributes (object)
+          + id: 49990 (number)
+          + due_date: 1475163480 (number) - Unix timestamp of the due date of the call
+          + due_date_formatted: `29/09/2016` (string) - Formatted due date of the call
+          + user_id: 40 (number) - The user linked to the call
+          + date_hour: `17:38` (string) - The hour when the call is scheduled
+          + description: Hfjf (string) - Description of the call
```
Comment (steftrenson) [inline src/calls.apib]: fcxds ;)

File: src/calls.apib [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+          + custom_fields (array)
+              + (object)
+                  + 1234: België (string) - Array of custom fields values keyed on custom_field_id
+          + call_outcome_name: Succesfull call (string) - The outcome of the call
```
Comment (steftrenson) [inline src/calls.apib]: Q: is this an enum, as in, can this have a limited number of possible strings?

**Comment by steftrenson** [comment]
Also, please resolve conflicts on apiary.apib

File: src/calls.apib (hunk 3 above)
Comment (svenlombaert) [inline src/calls.apib]: Nope, you can add as many outcomes as you want in the HR module

File: src/calls.apib (hunk 1 above)
Comment (svenlombaert) [inline src/calls.apib]: Impossible on a plain body, and we need this since it's not an attribute

File: src/calls.apib (hunk 3 above)
Comment (steftrenson) [inline src/calls.apib]: Ah, that's true, forgot about it.

File: apiary.apib [hunk 4]
```diff
@@ -2500,6 +2498,89 @@ Update a Teamleader meeting
...
+
+Add single callback.
+
++ Request (application/json)
+
+    + Attributes (object)
+        + due_date: `554774400` (number, required) - Due date of the call as Unix timestamp (seconds).
+        + hour: `14h00` (string, required) - Time of the call (hours and minutes). In the format 14h00.
+        + user_id: `1` (number, required) - The identifier of the user.
+        + for: `company` (enum[string]) - The entity to call.
```
Comment (Jdww) [inline apiary.apib]: for is also required

File: src/calls.apib [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+
+### /getCalls.php [POST /getCalls.php]
+
+Search calls
+
++ Request (application/json)
+    
+    + Attributes (object)
+        + amount: 10 (number, required) - The amount of calls returned per request (1-100)
```
Comment (wouterds) [inline src/calls.apib]: Not really an issue, but in the request above you used backticks on for the number fields (and here you don't).

**Pull request closed without merging**